includes/class-freemius.php

```diff
@@ -2717,7 +2710,7 @@ function is_addon_activated( $addon_id ) {
 		 * @return bool
 		 */
 		function is_addon_connected( $addon_id ) {
-			$sites = self::get_all_sites( Freemius::MODULE_TYPE_PLUGIN );
+			$sites = self::get_all_sites( $this->_module_type );
```

Comment:
@fajardoleo In the current case, since add-ons are always plugins, it should be `Freemius::MODULE_TYPE_PLUGIN`.